Sources/KeyboardSupport/KeyboardToolbar.swift

```diff
@@ -8,6 +8,24 @@
 
 import UIKit
 
+/// Represents a navigation type that a button could have on `KeyboardToolbar`.
+public enum KeyboardToolbarButtonNavigationType {
```

Comment:
What are your thoughts on moving this into the `KeyboardToolbar` namespace?
```swift
open class KeyboardToolbar: UIToolbar, KeyboardAccessory {
    
    // MARK: - Sub-types
    
    enum ButtonNavigationType {
        case back
        case next
        case done
        
        // ...
    }

    // ...
}
```

Reply:
I'm fine with that.